Hardware/HDD/SSDSamsung.cs

```diff
@@ -24,24 +24,21 @@ internal class SSDSamsung : AbstractHarddrive {
       new SmartAttribute(0x0C, SmartNames.PowerCycleCount, RawToInt),
       new SmartAttribute(0xAF, SmartNames.ProgramFailCountChip, RawToInt),
       new SmartAttribute(0xB0, SmartNames.EraseFailCountChip, RawToInt),
-      new SmartAttribute(0xB1, SmartNames.WearLevelingCount, RawToInt),
+      // Using 0xB1 because that is Wear Leveling Count attribute according to datasheet
+      new SmartAttribute(0xB1, SmartNames.WearLevelingCount, null, SensorType.Level, 0, SmartNames.RemainingLife),
       new SmartAttribute(0xB2, SmartNames.UsedReservedBlockCountChip, RawToInt),
       new SmartAttribute(0xB3, SmartNames.UsedReservedBlockCountTotal, RawToInt),
-
-      // Unused Reserved Block Count (Total)
-      new SmartAttribute(0xB4, SmartNames.RemainingLife,

```

Comment:
What's wrong with this one?

Reply:
That's odd to relate on total unused reserved blocks when there literally wear indicator in 0xB1 and that's official internal documentation.